fixtures/scripts/_head.sh

```diff
@@ -67,3 +67,11 @@ function halt_nodes {
   done
 }
 
+function check_old_version_uninstalled {
```

Comment:
@emilianodellacasa why this code is in the `head` if it's used only in one script?

Reply:
@yegor256 Ok, removed the function